src/components/toggleHiddenBox/ToggleHiddenBox.js

```diff
@@ -0,0 +1,12 @@
+import './toggle-hidden-box.scss';
+
+export const ToggleHiddenBox = (props) => {
+  const { visibleBoxStatus, clickHandler } = props;
+
+  return (
+    <div className="flex-container align-items-center">
+      <button type="button" onClick={() => clickHandler()}>{visibleBoxStatus ? 'Hide' : 'Show'}</button>&nbsp;&nbsp;
```

Comment:
why `visibleBoxStatus` and `clickHandler` are taken from props? It is inner state of component, it should be inside component.